### DISCUSSION TITLE
setTimeout handlers on module load

### DISCUSSION THREAD

**Discussion by pgg** [vladmandic/human#251] [Ideas]
zone js in angular reports [Violation] 'setTimeout' handler took <N>ms for every model loaded in the browser, similar to this post: [https://stackoverflow.com/questions/44060131/zone-js-violation-warnings-on-console-in-angular-project-only-on-chrome](url), maybe preventDefault could help?

**Comment by vladmandic** [comment]
anytime async function uses too much time, browser reports this. and loading of anything larger than few kb will always take more than few ms.  

preventDefault would help if there was some other registered handler for a handler, but this is not happening inside a handler, this is happening inside explicit load. there is nothing that can done about it, sorry.

**Reply by pgg** [reply]
ok
